src/openforms/emails/digest.py

```diff
@@ -179,3 +190,36 @@ def collect_broken_configurations() -> list[BrokenConfiguration]:
         )
 
     return broken_configurations
+
+
+def collect_invalid_certificates() -> list[InvalidCertificate]:
+    today = timezone.now()
+
+    invalid_certs = []
+    for cert in Certificate.objects.all():
```

Comment:
on our test env we have some certificates that are not actually used. I'd really restrict this to certificates that are configured on a zgw-consumers service (or SoapService or other place) to only report on certificates that are in use.

Even better would be to only consider services that are actually used, but I think that's a lot harder :thinking: